From 13cce78bcafd2aa22796e66e8a9898dc7407e9f0 Mon Sep 17 00:00:00 2001
From: Sameer Sharma <sameerx.sharma@intel.com>
Date: Mon, 18 Jun 2018 19:54:54 +0530
Subject: [PATCH 0706/1132] ASoC: Intel: Skylake: Enable use_tplg_pcm flag for
 BXTP platform

Enabling use_tplg_pcm flag to check whether FE dais will be
registered from topology during dai driver component registration
for BXTP platform

Change-Id: I60766088a56922e61c8010b0a587452db3db42d2
Signed-off-by: Sameer Sharma <sameerx.sharma@intel.com>
Reviewed-on:
Reviewed-by: Periyasamy, SriramX <sriramx.periyasamy@intel.com>
Reviewed-by: Gogineni, GiribabuX <giribabux.gogineni@intel.com>
Reviewed-by: Prabhu, PuneethX <puneethx.prabhu@intel.com>
Reviewed-by: Bozek, DominikX <dominikx.bozek@intel.com>
Reviewed-by: Lewandowski, Gustaw <gustaw.lewandowski@intel.com>
Tested-by: Madiwalar, MadiwalappaX <madiwalappax.madiwalar@intel.com>
---
 sound/soc/intel/skylake/skl.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/sound/soc/intel/skylake/skl.c b/sound/soc/intel/skylake/skl.c
index 7b3faf8..3687668 100644
--- a/sound/soc/intel/skylake/skl.c
+++ b/sound/soc/intel/skylake/skl.c
@@ -1188,6 +1188,10 @@ static struct skl_machine_pdata cnl_pdata = {
 	.use_tplg_pcm = true,
 };
 
+static struct skl_machine_pdata bxt_pdata = {
+       .use_tplg_pcm = true,
+};
+
 static struct snd_soc_acpi_mach sst_skl_devdata[] = {
 	{
 		.id = "INT343A",
@@ -1248,12 +1252,14 @@ IS_ENABLED(CONFIG_SND_SOC_INTEL_BXTP_IVI_GENERIC_MACH)
 		.id = "INT34C3",
 		.drv_name = "bxt_tdf8532",
 		.fw_filename = "intel/dsp_fw_bxtn.bin",
+		.pdata = &bxt_pdata,
 	},
 #elif IS_ENABLED(CONFIG_SND_SOC_INTEL_BXT_ULL_MACH)
 	{
 		.id = "INT34C3",
 		.drv_name = "bxt_ivi_ull",
 		.fw_filename = "intel/dsp_fw_ull_bxtn.bin",
+		.pdata = &bxt_pdata,
 	},
 #endif
 	{}
-- 
2.7.4

